Formula/git-cola.rb

```diff
@@ -15,21 +17,26 @@ class GitCola < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "7b7d769c22c41700d9813222188aca1bfe1f0971526590ad98da4c666059e0b4"
   end
 
-  depends_on "sphinx-doc" => :build
   depends_on "pyqt@5"
-  depends_on "python@3.9"
+  depends_on "python@3.10"
+
+  resource "packaging" do
+    url "https://files.pythonhosted.org/packages/df/9e/d1a7217f69310c1db8fdf8ab396229f55a699ce34a203691794c5d1cad0c/packaging-21.3.tar.gz"
+    sha256 "dd47c42927d89ab911e606518907cc2d3a1f38bbd026385970643f9c5b8ecfeb"
+  end
 
-  uses_from_macos "rsync"
+  resource "pyparsing" do
+    url "https://files.pythonhosted.org/packages/71/22/207523d16464c40a0310d2d4d8926daffa00ac1f5b1576170a32db749636/pyparsing-3.0.9.tar.gz"
+    sha256 "2b020ecf7d21b687f219b71ecad3631f644a47f01403fa1d1036b0c6416d70fb"
+  end
+
+  resource "QtPy" do
+    url "https://files.pythonhosted.org/packages/4f/8b/2830a11f0773dded3bb9943daeed7c1eefb214f84d72a213a5d4681430b2/QtPy-2.1.0.tar.gz"
+    sha256 "ca8cd4217175186344299ee4c0f7e7adcf362c70852ba35b255a534077025c06"
+  end
 
   def install
-    # setuptools>=60 prefers its own bundled distutils, which breaks the installation
-    # Remove when disutils is no longer used: https://github.com/git-cola/git-cola/issues/1201
-    ENV["SETUPTOOLS_USE_DISTUTILS"] = "stdlib"
-
-    ENV.delete("PYTHONPATH")
-    ENV["PYTHON"] = which("python3")
-    system "make", "prefix=#{prefix}", "install"
-    system "make", "install-doc", "prefix=#{prefix}", "SPHINXBUILD=#{Formula["sphinx-doc"].opt_bin}/sphinx-build"
```

Comment:
The docs are a pain to build. They now require `jaraco.packaging` and `rst.linker`, but this would make maintenance trickier unless we install them into venv. So, either we install unneeded packages or we drop docs (html/man).